plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -187,7 +181,12 @@ export class PgInstrumentation extends InstrumentationBase {
               params
             );
           } else {
-            span = utils.handleTextQuery.call(this, plugin.tracer, query);
+            span = utils.handleTextQuery.call(
+              this,
+              plugin.tracer,
+              plugin.getConfig() as PgInstrumentationConfig,
```

Comment:
As I was going over the code in this instrumentation, I noticed that the config requires casting here, we could remove that need by doing something similar to some of the other instrumentations - https://github.com/open-telemetry/opentelemetry-js-contrib/blob/6684b56b8043f094b95fc3c1ce5e5599e694bad4/plugins/node/opentelemetry-instrumentation-koa/src/instrumentation.ts#L50-L56

This would remove some annoying type casts and replace it with a singular one. Not super necessary to be done, but would be nice to clean up the codebase a bit?